src/client/utils/forestEstimations.ts

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios'
+
+import { ForestEstimations, ForestEstimationsData, ForestSource, sourcesMetadata } from '@meta/geo'
+import { hansenPercentages } from '@meta/geo/forest'
+
+/**
+ * Type for tabular data, consisting of an array of arrays with
+ * source, area, and coverage percentage.
+ */
+export type TabularForestEstimationData = {
+  data: [string, number, number][]
+  fra1ALandArea: number // fra1ALandArea reported in thousands of Ha.
+  fra1aForestArea: number // fra1aForestArea reported in thousands of Ha.
+}
+
+/**
+ * Makes an API call to get the Forest Estimations in a country in a year,
+ * in a tabular way.
+ *
+ * @param {string} countryIso The country to query the statistics data.
+ * @param {string} year The year to query the statistics data.
+ * @public
+ */
+export const getForestEstimationData = async (
```

Comment:
this is not really a global utility. can u for now just move it where it's used?

Reply:
Sure. I moved it to a utils folder under Geo, similar to what we do in other folders.